### ISSUE TITLE
Test accessibility of Github browser/desktop/app with screen reader

### ISSUE THREAD

**Issue by seekanddefine** [knowledge-for-good/knowledgeforgood#305]
How accessible is Github with a screen reader on different platforms?

Explore together if it is possible to navigate each of these platforms with a screen reader.

*   [ ] browser
*   [ ] desktop
*   [ ] app

**Comment by seekanddefine** [comment]
We met yesterday. Lorelei had already explored the browser which was initially quite inaccessible, so she downloaded the mobile app which was more approachable. For now we have decided to use a combination of issues on Github and Trello because the mobile app doesn't have access to project boards.
